README.rst

```diff
@@ -137,9 +137,6 @@ You may consult our `online documentation <https://diffpy.github.io/diffpy.pdfmo
 USING PDFmorph
 ------------------------------------------------------------------------
 
-For detailed instructions and full tutorial, consult the user manual
```

Comment:
I think we still would like a link to the manual.  So this shouldn't be deleted but modified to point to where the new manual will be.

If we don't quite know that yet, please create an issue to fix that later, then we can resolve this conversation.  But if possible, fix it.